src/formats/AmberNetCDF.cpp

```diff
@@ -114,12 +126,45 @@ void AmberNetCDFFormat::read_step(const size_t step, Frame& frame) {
     }
 }
 
-void AmberNetCDFFormat::read(Frame& frame) {
+template <AmberFormat F>
+void Amber<F>::read(Frame& frame) {
+    if (FormatSpec<F>::one_frame && step_ != 0) {
```

Comment:
Instead of having the `one_frame` member, you could do something like 
```suggestion
    if (F ==  NCRST && step_ != 0) {
```

Reply:
That's too easy :-)